lib/improver/cli/estimate_emos_coefficients.py

```diff
@@ -69,12 +74,25 @@ def main(argv=None):
                              'This cycletime is in the format '
                              'YYYYMMDDTHHMMZ.')
     # Filepaths for historic and truth data.
-    parser.add_argument('historic_filepath', metavar='HISTORIC_FILEPATH',
+    parser.add_argument('--historic_filepath', metavar='HISTORIC_FILEPATH',
                         help='A path to an input NetCDF file containing the '
                              'historic forecast(s) used for calibration.')
-    parser.add_argument('truth_filepath', metavar='TRUTH_FILEPATH',
+    parser.add_argument('--truth_filepath', metavar='TRUTH_FILEPATH',
                         help='A path to an input NetCDF file containing the '
                              'historic truth analyses used for calibration.')
+    # Input filepaths
+    parser.add_argument('--combined_filepath', metavar='COMBINED_FILEPATH',
+                        help='The path to the input NetCDF files containing '
```

Comment:
This is actually a list of several path**s**? Should it have an nargs='+' type thing going on? Same for the truth and historic filepath arguements.

Reply:
I've added `nargs='+'` as suggested.